applications/doodle/src/doodle.erl

```diff
@@ -24,6 +25,16 @@
 start_link() ->
     _ = start_deps(),
     _ = declare_exchanges(),
```

Comment:
Are you trying to update the existing doodle config doc with the reschedule object?
Why are you adding an extra document `reschedule.json` rather than updating the existing doodle config document?